llama_stack/apis/inference/inference.py

```diff
@@ -1157,7 +1157,8 @@ async def embeddings(
         """
         ...
 
-    @webmethod(route="/inference/rerank", method="POST", experimental=True, level=LLAMA_STACK_API_V1)
+    @webmethod(route="/inference/rerank", method="POST", deprecated=True, level=LLAMA_STACK_API_V1)
```

Comment:
this isn't deprecated

Reply:
Hey we are moving to 0.3.0 coming week and going to land all kinds of backwards incompatible changes. For this window, let's just not have this deprecation then?

Reply:
@ashwinb , this `deprecated=True` doesn't actually get rid of the route, but instead just lets the users know that they should opt to use the `v1alpha` version. So this doesn't really break anything it just aligns us more with the API leveling doc.

Reply:
we are going to keep the deprecated ones around until 0.4

Reply:
> @ashwinb , this `deprecated=True` doesn't actually get rid of the route, but instead just lets the users know that they should opt to use the `v1alpha` version. So this doesn't really break anything it just aligns us more with the API leveling doc.

yes I understood that. However I am saying we don't need this for this particular change because we are doing all sorts or removals _this week_. So might as well not keep the old route hanging.

Reply:
Ah ok @ashwinb that makes sense. I am fine with either keeping this around with deprecated or just deleting it. Let's see what @mattf thinks!

Reply:
go for it